frontend_build/src/parse_bundle_plugin.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
```

Comment:
Ha, didn't fully think that through :)

I made a few changes:
* moved the input validation to the top
* switched some `_.extend` code on the bundle to simple assignment - makes it clear that the properties were not also being partially set elsewhere
* moved some loader code to webpack.config.base